### PR TITLE
Add a banner to RH/deb pages about GPG key changes

### PR DESCRIPTION
**Pull request by NotMyFault**
Ref https://github.com/jenkins-infra/helpdesk/issues/3457

Screenshot 2023-03-27 at 16 55 19

Thoughts?

### CODE REVIEW

File: templates/header.debian.html [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 {% block distribution_instruction %}
 
+<div class="alert alert-danger">
+    <b>Important Notice:</b> Beginning with LTS 2.387.2 and weekly 2.397, releases will be signed with an updated GPG key. <br>
```
Comment (dduportal) [inline templates/header.debian.html:6]:
```suggestion
    <b>Important Notice:</b> Beginning with LTS 2.387.2 and weekly 2.397, releases will be signed with a new GPG key. <br>
```

File: templates/header.redhat.html [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 {% block distribution_instruction %}
 
+<div class="alert alert-danger">
+    <b>Important Notice:</b> Beginning with LTS 2.387.2 and weekly 2.397, releases will be signed with an updated GPG key. <br>
```
Comment (dduportal) [inline templates/header.redhat.html:6]:
```suggestion
    <b>Important Notice:</b> Beginning with LTS 2.387.2 and weekly 2.397, releases will be signed with a new GPG key. <br>
```

**Review by dduportal — COMMENTED** [review]
Non blocking proposal to materialize there is a new key version.

Also, we might want to add at least the new link if it is ok for you?

Something like this (as per the code below):

```
{{web_url}}/{{organization}}-2023.key
```

**Comment by NotMyFault** [comment]
> Non blocking proposal to materialize there is a new key version.

Oic, I didn't check out the puppet PR yet. Nice idea with the version suffix.

**Review by MarkEWaite — APPROVED** [review]

**Comment by MarkEWaite** [comment]
I assume that we can make this visible as soon as https://pkg.jenkins.io/debian/jenkins.io-2023.key and friends are visible to users.  Tuesday morning European time seems soon enough to me.

**Pull request merged**